Code/GraphMol/FMCS/testFMCS_Unit.cpp

```diff
@@ -565,9 +564,9 @@ void testJnk1LigandsDistance(){
   const char* jnk1sdf = "/Code/GraphMol/FMCS/testData/Jnk1_ligands.sdf";
   std::string fn(rdbase + jnk1sdf);
 
-  RDKit::MolSupplier* suppl = nullptr;
+  std::unique_ptr<RDKit::MolSupplier> suppl;
   try {
-    suppl = new RDKit::SDMolSupplier(fn);
+    suppl.reset(new RDKit::SDMolSupplier(fn));
   } catch (...) {
     std::cerr << "ERROR: RDKit could not load input file" << std::endl;
```

Comment:
feels like this actually hides the error from SDMolSupplier, and doesn't print the filename it tried for anyway...

Maybe:

    RDKit::SDMolSupplier suppl(fn);

Reply:
I'm not sure I agree with that, particularly not in a test file (which this is).
It's also arguably outside the scope of this PR to refactor the tests themselves.